PurchasesCoreSwiftTests/Purchasing/StoreKitWrapperTests.swift

```diff
@@ -220,7 +220,7 @@ class StoreKitWrapperTests: XCTestCase, RCStoreKitWrapperDelegate {
 
             let mockProduct = MockSKProduct(mockProductIdentifier: productId)
             let mockDiscount = MockPaymentDiscount(mockIdentifier: discountId)
-            let payment = wrapper.payment(with: mockProduct, discount: mockDiscount)
+            let payment = wrapper.payment(withProduct: mockProduct, discount: mockDiscount)
```

Comment:
slight naming change for swift only. I can use `@objc` to keep the old interface. This one doesn't have a native `StoreKit` equivalent, so we should do whatever we like the most. 
I kinda like `withProduct`, but don't have a strong preference.

Reply:
I'm not opposed to this change, I think it's better. Now is the time to do updates like this IMO.